git/windows.xml

```diff
@@ -271,12 +213,26 @@
     <needs-terminal/>
     <name>scp</name>
     <summary>Secure remote file copy</summary>
-  </entry-point>
-  <entry-point command="curl">
+  </entry-point><!--<entry-point command="curl">
     <needs-terminal/>
     <name>cURL</name>
     <summary>A comand-line client for HTTP, FTP, etc.</summary>
   </entry-point>
+  <entry-point command="gpg" binary-name="gpg">
```

Comment:
Zero Install for Windows offers to create command-line aliases for all `<entry-point>`s with `<needs-terminal/>` by default.

I thought the entry points for cURL and GnuPG might be useful, but they are in conflict with other feeds offering these tools stand-alone (http://repo.roscidus.com/security/gnupg and http://repo.roscidus.com/utils/curl).

I was undecided on whether these entry points should be in the feed or not, so I left them in as comments for possible future reference.